metadata/picture_set/__init__.py

```diff
@@ -0,0 +1,35 @@
+from pydantic import BaseModel
+from datetime import date
+from metadata import validator
+
+def build_picture_set(user_id:str,nb_picture:int):
+    """
+    This function builds the picture_set needed to represent each folder (with pictures in it) of a container.
+    
+    Parameters:
+    - user_id (str): The UUID of the user uploading.
+    - nb_picture (int): The amount of picture present in the folder.
+
+    Returns:
+    - The picture_set in .
```

Comment:
in ... after the pause we will know the mistery return type of picture_set :laughing: